docs/actors/development/state_persistence.md

```diff
@@ -20,22 +20,20 @@ A migration is when a process running on a server has to stop and move to anothe
 
 **When a migration event occurs, you only have a few seconds to save your work.**
 
-## [](#why-migrations-happen)Why do migrations happen?
+## [](#why-do-migrations-happen)Why do migrations happen?
 
-Migrations happen because of the availability of Amazon EC2's **spot instances**. [Visit Amazon for more information](https://aws.amazon.com/ec2/spot/?cards.sort-by=item.additionalFields.startDateTime&cards.sort-order=asc).
+- To optimize server workloads.
+- When a server crashes (unlikely).
+- During deploys of new [worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) software to the server.
```

Comment:
It's actually [this worker](https://github.com/apify/apify-worker), not a web worker, that we deploy :) 

Maybe we can just say "When we release new features and fix bugs."

Reply:
Yep, that's way better :D